### PR TITLE
Fixed dates and clarifed the role of UAegean in wallet ecosystem

### PR DESCRIPTION
**Pull request by endimion**
also update the role of UAegean in the rest of the RFCs...

### CODE REVIEW

File: ewc-rfc001-issue-verifiable-credential.md [hunk 1]
```diff
@@ -878,11 +878,11 @@ The table below summarises the success/error responses that can be used:
    </td>
   </tr>
   <tr>
-   <td>UAegean 
+   <td>UAegean on behalf of CFF
```
Comment (lalc) [inline ewc-rfc001-issue-verifiable-credential.md:881]: You could exapand on CFF?

File: ewc-rfc100-interoperability-profile-towards-itb-v1.0.md [hunk 2]
```diff
@@ -325,11 +325,12 @@ The Verifier service MUST follow the EWC RFC [4] in its implementation.
    </td>
   </tr>
   <tr>
-   <td>UAegean 
+   <td>UAegean on behalf of CFF
```
Comment (lalc) [inline ewc-rfc100-interoperability-profile-towards-itb-v1.0.md:328]: Expand CFF

File: ewc-rfc100-interoperability-profile-towards-itb-v1.0.md [hunk 3]
```diff
@@ -379,11 +380,11 @@ The Verifier service MUST follow the EWC RFC [4] in its implementation.
...
```
Comment (lalc) [inline ewc-rfc100-interoperability-profile-towards-itb-v1.0.md:383]: Expand CFF

**Review by lalc — COMMENTED** [review]
Looks good to me. Pls fix the comments + please check the merge conflicts.

**Comment by endimion** [comment]
ok cff acronym expanded... please merge when able  :)

**Review by lalc — APPROVED** [review]

**Pull request merged**